src/main/java/bowling/domain/Player.java

```diff
@@ -0,0 +1,35 @@
+package bowling.domain;
+
+import java.util.Objects;
+
+public class Player {
+    private final String player;
+
+    private Player(String player) {
+        this.player = player;
+    }
+
+    public static Player from(String player){
```

Comment:
```suggestion
    public static Player from(String player) {
```

매의 눈 @_@
또한 Players 내에 존재하는 검증로직을 해당 객체의 역할로 넘겨주면 어떨까요!?

Reply:
각각의 Player에서 validation을 해주는게 맞는 것 같습니다!